packages/amplify-codegen/tests/commands/statements.test.js

```diff
@@ -87,7 +87,7 @@ describe('command - statements', () => {
     await generateStatements(MOCK_CONTEXT, forceDownload);
     expect(getFrontEndHandler).toHaveBeenCalledWith(MOCK_CONTEXT);
     expect(loadConfig).toHaveBeenCalledWith(MOCK_CONTEXT, false);
-    expect(getRelativeTypesPath).toHaveBeenCalledWith('MOCK_PROJECT_ROOT/MOCK_STATEMENTS_PATH', 'API.TS');
+    expect(getRelativeTypesPath).toHaveBeenCalledWith(path.join(MOCK_PROJECT_ROOT, MOCK_STATEMENTS_PATH), 'API.TS');
```

Comment:
It doesn't matter if `getRelativeTypePath` is called with a path with posix or windows separators.